src/js/yeldaWebchatInjector.js

```diff
@@ -949,6 +965,48 @@ class YeldaChat {
       webchatFrame.contentWindow.postMessage({ event: 'sendUserMessage', data: message })
     }
   }
+
+  /**
+   * Listen leave viewport event handling
+   * @param {Boolean} [remove=false]
+   */
+  listenLeaveViewport(remove = false) {
+    const md = new MobileDetect(navigator.userAgent)
+    const isMobile = md.mobile() !== null
+
+    // Not needed to listen on mobile
+    if (isMobile) {
+      return
+    }
```

Comment:
We **don't need to** listen to the event on mobile, or we **cannot** do it? 
Try to explain a little bit more, why we don't need to listen on mobile.